test/unit/logic/vote.js

```diff
@@ -378,7 +378,7 @@ describe('vote', () => {
 	});
 
 	describe('verifyVote', () => {
-		it('should throw if vote is of invalid length', done => {
+		it('should return error if vote is of invalid length', done => {
```

Comment:
The description and expected error doesn't match, it should be `should return error if vote is of invalid format`

Reply:
the test description doesn't say what the error will be. The test description specifies what the kind of input we pass to the function.